src/samples/provision-chromeapp/background.js

```diff
@@ -0,0 +1,37 @@
+chrome.app.runtime.onLaunched.addListener(function() {
+  // Prevent the app from suspending.
+  // TODO: Use persistent sockets instead:
+  //       https://github.com/uProxy/uproxy/issues/1746
+  chrome.app.window.create('index.html', {
+  	id: "provision",
+    outerBounds: {
+      width: 200,
+      height: 200,
+    }
+  });
+
+  var script = document.createElement('script');
+  script.src = 'freedom-for-chrome/freedom-for-chrome.js';
+  document.head.appendChild(script);
+  script.onload = function() {
+    console.log('loading freedom!');
+    freedom('uproxy-lib/provision/freedom-module.json', {
+      'logger': 'uproxy-lib/loggingprovider/freedom-module.json',
+      'debug': 'debug'
+    }).then(function(moduleFactory) {
+      var module = moduleFactory();
```

Comment:
module, i think, is becoming a reserved word in js. maybe pick a different symbol?